docs/developer/node/node-rpc.md

```diff
@@ -132,9 +132,13 @@ You can specify the maximum fee limit for the transaction `maxFee`.
 All addresses with balances less than dust are cleaned every time a new epoch starts. Dust coins are burnt to prevent spam and minimize the size of the blockchain state. You can calculate the dust size using the following formula:
 
 ```
-0.01/networkSize
+bcn_feePerGas * 1000

```

Comment:
The proposed formula is incorrect. However the previous formula isn't correct either. 
The right one is `10/networkSize`. So the dust amount for the current network 9922 is around 0.001

Reply:
I've updated it.